src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -44,15 +45,13 @@ public void setBrowserType(String browserType) {
     }
 
     /**
-     *
      * @param browserType the browser for the run
      */
     public void setBrowserType(BrowserType browserType) {
         this.browserType = browserType;
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes

Reply:
added the blank line